felles/abac/src/main/java/no/nav/vedtak/sikkerhet/abac/PepImpl.java

```diff
@@ -29,69 +26,68 @@ public class PepImpl implements Pep {
     private PdpKlient pdpKlient;
     private PdpRequestBuilder builder;
 
-    private Set<String> pipUsers;
-    private TokenProvider tokenProvider;
     private String preAuthorized;
     private Cluster residentCluster;
     private String residentNamespace;
 
-    public PepImpl() {
+    PepImpl() {
+        // CDI proxy
     }
 
     @Inject
-    public PepImpl(PdpKlient pdpKlient,
-                   TokenProvider tokenProvider,
-                   PdpRequestBuilder pdpRequestBuilder,
-                   @KonfigVerdi(value = "pip.users", required = false) String pipUsers) {
+    public PepImpl(PdpKlient pdpKlient, PdpRequestBuilder pdpRequestBuilder) {
         this.pdpKlient = pdpKlient;
         this.builder = pdpRequestBuilder;
-        this.tokenProvider = tokenProvider;
-        this.pipUsers = konfigurePipUsers(pipUsers);
         this.preAuthorized = ENV.getProperty(AzureProperty.AZURE_APP_PRE_AUTHORIZED_APPS.name()); // eg json array av objekt("name", "clientId")
         this.residentCluster = ENV.getCluster();
         this.residentNamespace = ENV.namespace();
     }
 
-    protected Set<String> konfigurePipUsers(String pipUsers) {
-        if (pipUsers != null) {
-            return Set.of(pipUsers.toLowerCase().split(","));
-        }
-        return Set.of();
-    }
-
     @Override
     public Tilgangsbeslutning vurderTilgang(BeskyttetRessursAttributter beskyttetRessursAttributter) {
         var appRessurser = builder.lagAppRessursData(beskyttetRessursAttributter.getDataAttributter());
 
-        if (PIP.equals(beskyttetRessursAttributter.getResourceType())) {
-            return vurderTilgangTilPipTjeneste(beskyttetRessursAttributter, appRessurser);
-        }
-        if (kanForetaLokalTilgangsbeslutning(beskyttetRessursAttributter)) {
-            return new Tilgangsbeslutning(harTilgang(beskyttetRessursAttributter) ? GODKJENT : AVSLÅTT_ANNEN_ÅRSAK, beskyttetRessursAttributter, appRessurser);
+        if (skalVurdereLokalTilgang(beskyttetRessursAttributter)) {
+            return vurderLokalTilgang(beskyttetRessursAttributter, appRessurser);
```

Comment:
Kan hende det blir et loop-hole for OBO-kall på PIP? Noe slikt burde være safe
if (kanForetaLokalTilgangsbeslutning()) return vurderLokalTilgang
else if (PIP) return AVSLÅTT
else kall pdpklient

Reply:
Tror de blir bedre nå